Pins-Examples/IT81202B_Pinout_Example_Option1.json

```diff
@@ -0,0 +1,79 @@
+{
+    "partType": "EC",
+    "manufacturer": "ITE",
+    "mpn": "IT81202E",
+    "datasheetVersion": "C0.3.2",
+    "releaseDate": null,
+    "pins": [
+        {
+            "_comment_1_ForWG":" Option1 : Use Arrays",
+            "terminal_identifier": "5",
```

Comment:
"functionProperities": [
{
"perFunctionName": "ALERT#",
"function": "SPI",
"direction": "inout", 
"polarity": "low"
},
{
"perFunctionName": "SERIRQ",
"function": "IRQ",
"direction": "in", 
"polarity": "high"
},
{
"perFunctionName": "GPM6",
"function": "GPIO",
"direction": "in", 
"polarity": "high"
}
]